components/worktemplates/components/preview/accordion.tsx

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Could this customization just be available in the original component? Maybe controlled through props?

Reply:
I'm unsure if applying this style to the main component behind a prop is bringing any benefits 🤔 It's just forcing the generic component to be responsible for the specific style of this modal